### PR TITLE
Cup and Die implementation (w/ tests)

### PR DESCRIPTION
**Pull request by StephaneB1**

### CODE REVIEW

**Review by LordTT — APPROVED** [review]

File: src/main/java/Die.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+public class Die {
+
+    private static int TOTAL_FACES = 6;
+
+    int faceValue = -1;
+
+    public Die() {}
+
+    public void roll() {
```
Comment (KurohanaJuri) [inline src/main/java/Die.java:13]: Static methode may be?

File: src/test/java/CupTest.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    @Test
+    void negativeNumberOfDice() {
+        try{
+            Cup cup = new Cup(-1);
+            fail("Exception not thrown");
+        }catch(Exception e){
+            // OK an exception has been thrown
+        }
+    }
```
Comment (KurohanaJuri) [inline src/test/java/CupTest.java]: If you want to test if it thow a exception try this :

```suggestion
    @Test
    void negativeNumberOfDice() {
        Assertions.assertThrows(RuntimeException.class, ()->
                Cup cup = new Cup(-1)
        );
    }
```

File: src/test/java/DieTest.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    @Test
+    void testFaceValueNotRolled() {
+        try{
+            Die die = new Die();
+            die.getFaceValue();
+            fail("Exception not thrown");
+        }catch(Exception e){
+            // OK an exception has been thrown
+        }
+    }
```
Comment (KurohanaJuri) [inline src/test/java/DieTest.java]: Same review that in CupTest

**Review by KurohanaJuri — CHANGES_REQUESTED** [review]

File: src/main/java/Die.java (hunk 1 above)
Comment (StephaneB1) [reply src/main/java/Die.java:13]: I wouldn't since the faceValue isn't static

File: src/test/java/CupTest.java (hunk 2 above)
Comment (StephaneB1) [reply src/test/java/CupTest.java]: Got it, changing now

**Review by KurohanaJuri — APPROVED** [review]

**Pull request merged**